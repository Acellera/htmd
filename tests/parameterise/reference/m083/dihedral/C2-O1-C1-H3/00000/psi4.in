set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 -2.240291	 0.559484	 0.604969
C	 -2.008299	 -0.205926	 -0.129530
O	 -0.774054	 0.001095	 -0.743181
H	 -2.752932	 -0.160639	 -0.912984
H	 -2.076115	 -1.178578	 0.348530
C	 -0.520590	 1.222591	 -1.414175
C	 0.976003	 1.166795	 -1.712687
C	 -1.311586	 1.308306	 -2.723437
C	 -0.829324	 2.422663	 -0.513271
H	 1.201904	 0.303346	 -2.328200
H	 1.537702	 1.078299	 -0.789628
H	 1.304202	 2.060079	 -2.234214
H	 -1.142254	 0.420237	 -3.323179
H	 -0.995698	 2.173244	 -3.297629
H	 -2.378141	 1.405646	 -2.551564
H	 -0.320934	 2.317681	 0.439393
H	 -0.490512	 3.341127	 -0.981419
H	 -1.891946	 2.528947	 -0.323088

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		2 3 6 7
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "18  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
