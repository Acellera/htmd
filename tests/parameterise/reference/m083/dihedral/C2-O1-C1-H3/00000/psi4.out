    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 16:42:02 2016

    Process ID:   8578
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 -2.240291	 0.559484	 0.604969
C	 -2.008299	 -0.205926	 -0.129530
O	 -0.774054	 0.001095	 -0.743181
H	 -2.752932	 -0.160639	 -0.912984
H	 -2.076115	 -1.178578	 0.348530
C	 -0.520590	 1.222591	 -1.414175
C	 0.976003	 1.166795	 -1.712687
C	 -1.311586	 1.308306	 -2.723437
C	 -0.829324	 2.422663	 -0.513271
H	 1.201904	 0.303346	 -2.328200
H	 1.537702	 1.078299	 -0.789628
H	 1.304202	 2.060079	 -2.234214
H	 -1.142254	 0.420237	 -3.323179
H	 -0.995698	 2.173244	 -3.297629
H	 -2.378141	 1.405646	 -2.551564
H	 -0.320934	 2.317681	 0.439393
H	 -0.490512	 3.341127	 -0.981419
H	 -1.891946	 2.528947	 -0.323088

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		2 3 6 7
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "18  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-1-4-3.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 16:42:02 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           H         -1.479335568729    -0.416300818250     1.800051896229     1.007825032070
           C         -1.247343568729    -1.181710818250     1.065552896229    12.000000000000
           O         -0.013098568729    -0.974689818250     0.451901896229    15.994914619560
           H         -1.991976568729    -1.136423818250     0.282098896229     1.007825032070
           H         -1.315159568729    -2.154362818250     1.543612896229     1.007825032070
           C          0.240365431271     0.246806181750    -0.219092103771    12.000000000000
           C          1.736958431271     0.191010181750    -0.517604103771    12.000000000000
           C         -0.550630568729     0.332521181750    -1.528354103771    12.000000000000
           C         -0.068368568729     1.446878181750     0.681811896229    12.000000000000
           H          1.962859431271    -0.672438818250    -1.133117103771     1.007825032070
           H          2.298657431271     0.102514181750     0.405454896229     1.007825032070
           H          2.065157431271     1.084294181750    -1.039131103771     1.007825032070
           H         -0.381298568729    -0.555547818250    -2.128096103771     1.007825032070
           H         -0.234742568729     1.197459181750    -2.102546103771     1.007825032070
           H         -1.617185568729     0.429861181750    -1.356481103771     1.007825032070
           H          0.440021431271     1.341896181750     1.634475896229     1.007825032070
           H          0.270443431271     2.365342181750     0.213663896229     1.007825032070
           H         -1.130990568729     1.553162181750     0.871994896229     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      0.14729  B =      0.09163  C =      0.09154 [cm^-1]
  Rotational constants: A =   4415.70584  B =   2747.14332  C =   2744.41779 [MHz]
  Nuclear repulsion =  272.459396202354696

  Charge       = 0
  Multiplicity = 1
  Electrons    = 50
  Nalpha       = 25
  Nbeta        = 25

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 60
    Number of basis function: 114
    Number of Cartesian functions: 114
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A        114     114       0       0       0       0
   -------------------------------------------------------
    Total     114     114      25      25      25       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 252
    Number of basis function: 786
    Number of Cartesian functions: 786
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 3.2381074168E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -178.67240323053022   -1.78672e+02   9.77431e-02 
   @DF-RHF iter   2:  -221.22660975489867   -4.25542e+01   4.69539e-02 DIIS
   @DF-RHF iter   3:  -240.99698873732325   -1.97704e+01   5.48710e-02 DIIS
   @DF-RHF iter   4:  -263.18314471729730   -2.21862e+01   3.52995e-02 DIIS
   @DF-RHF iter   5:  -268.22967277962960   -5.04653e+00   2.09746e-02 DIIS
   @DF-RHF iter   6:  -270.49070943865775   -2.26104e+00   1.00634e-02 DIIS
   @DF-RHF iter   7:  -271.11333277215624   -6.22623e-01   2.86701e-03 DIIS
   @DF-RHF iter   8:  -271.16693922895627   -5.36065e-02   8.27313e-04 DIIS
   @DF-RHF iter   9:  -271.17215961306567   -5.22038e-03   1.56266e-04 DIIS
   @DF-RHF iter  10:  -271.17240833916037   -2.48726e-04   5.78886e-05 DIIS
   @DF-RHF iter  11:  -271.17244715224399   -3.88131e-05   2.67462e-05 DIIS
   @DF-RHF iter  12:  -271.17245366484337   -6.51260e-06   5.30603e-06 DIIS
   @DF-RHF iter  13:  -271.17245407521375   -4.10370e-07   1.17976e-06 DIIS
   @DF-RHF iter  14:  -271.17245409244435   -1.72306e-08   3.65854e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -20.544713     2A    -11.282228     3A    -11.260767  
       4A    -11.209863     5A    -11.207963     6A    -11.203220  
       7A     -1.369366     8A     -1.072798     9A     -0.947052  
      10A     -0.939506    11A     -0.911714    12A     -0.751926  
      13A     -0.654351    14A     -0.645692    15A     -0.639108  
      16A     -0.592878    17A     -0.576702    18A     -0.549622  
      19A     -0.533669    20A     -0.525119    21A     -0.514736  
      22A     -0.489050    23A     -0.486681    24A     -0.437044  
      25A     -0.402065  

    Virtual:                                                              

      26A      0.219517    27A      0.264562    28A      0.278464  
      29A      0.287563    30A      0.306587    31A      0.310764  
      32A      0.321055    33A      0.340591    34A      0.352131  
      35A      0.355299    36A      0.362800    37A      0.369998  
      38A      0.391280    39A      0.404636    40A      0.426001  
      41A      0.443144    42A      0.476032    43A      0.702343  
      44A      0.712958    45A      0.730857    46A      0.769240  
      47A      0.770931    48A      0.813895    49A      0.821844  
      50A      0.880733    51A      0.904257    52A      0.941883  
      53A      0.952638    54A      0.976202    55A      1.011353  
      56A      1.074138    57A      1.102898    58A      1.121011  
      59A      1.142077    60A      1.157328    61A      1.171315  
      62A      1.186873    63A      1.192840    64A      1.207191  
      65A      1.214291    66A      1.225529    67A      1.228440  
      68A      1.263841    69A      1.274119    70A      1.291957  
      71A      1.312749    72A      1.443013    73A      1.452900  
      74A      1.616781    75A      1.649808    76A      1.658142  
      77A      1.676719    78A      1.724073    79A      1.862560  
      80A      1.946522    81A      2.020420    82A      2.024933  
      83A      2.045197    84A      2.107284    85A      2.178560  
      86A      2.187976    87A      2.253949    88A      2.306777  
      89A      2.401019    90A      2.409867    91A      2.455745  
      92A      2.483457    93A      2.501369    94A      2.523629  
      95A      2.587423    96A      2.594016    97A      2.612723  
      98A      2.622903    99A      2.632969   100A      2.695285  
     101A      2.776705   102A      2.798765   103A      2.895696  
     104A      2.962066   105A      3.088031   106A      3.120170  
     107A      3.279805   108A      3.341992   109A      4.396972  
     110A      4.638516   111A      4.692402   112A      4.739909  
     113A      4.760030   114A      5.092951  

    Final Occupation by Irrep:
              A 
    DOCC [    25 ]

  Energy converged.

  @DF-RHF Final Energy:  -271.17245409244435

   => Energetics <=

    Nuclear Repulsion Energy =            272.4593962023546965
    One-Electron Energy =                -907.7678792434794559
    Two-Electron Energy =                 364.1360289486799502
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -271.1724540924448092

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:    -1.0440      Y:     2.9385      Z:    -0.9428

  Electronic Dipole Moment: (a.u.)
     X:     0.6811      Y:    -2.5594      Z:     0.9368

  Dipole Moment: (a.u.)
     X:    -0.3628      Y:     0.3791      Z:    -0.0060     Total:     0.5248

  Dipole Moment: (Debye)
     X:    -0.9223      Y:     0.9635      Z:    -0.0152     Total:     1.3338


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-1-4-3.cx1.hpc.ic.ac.uk at Tue Jul  5 16:42:12 2016
Module time:
	user time   =       8.96 seconds =       0.15 minutes
	system time =       0.07 seconds =       0.00 minutes
	total time  =         10 seconds =       0.17 minutes
Total time:
	user time   =       8.96 seconds =       0.15 minutes
	system time =       0.07 seconds =       0.00 minutes
	total time  =         10 seconds =       0.17 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:    -1.0440      Y:     2.9385      Z:    -0.9428

  Electronic Dipole Moment: (a.u.)
     X:     0.6811      Y:    -2.5594      Z:     0.9368

  Dipole Moment: (a.u.)
     X:    -0.3628      Y:     0.3791      Z:    -0.0060     Total:     0.5248

  Dipole Moment: (Debye)
     X:    -0.9223      Y:     0.9635      Z:    -0.0152     Total:     1.3338

  Quadrupole Moment: (Debye Ang)
    XX:   -38.8043     YY:   -40.5819     ZZ:   -39.1969
    XY:     1.7343     XZ:    -1.4564     YZ:    -0.4943

  Traceless Quadrupole Moment: (Debye Ang)
    XX:     0.7234     YY:    -1.0542     ZZ:     0.3308
    XY:     1.7343     XZ:    -1.4564     YZ:    -0.4943

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     H     0.42865  0.42865  0.00000  0.14270
       2     C     3.07907  3.07907  0.00000 -0.15814
       3     O     4.32048  4.32048  0.00000 -0.64097
       4     H     0.43083  0.43083  0.00000  0.13833
       5     H     0.41220  0.41220  0.00000  0.17559
       6     C     2.84138  2.84138  0.00000  0.31724
       7     C     3.23250  3.23250  0.00000 -0.46501
       8     C     3.24485  3.24485  0.00000 -0.48969
       9     C     3.25181  3.25181  0.00000 -0.50362
      10     H     0.41374  0.41374  0.00000  0.17252
      11     H     0.41444  0.41444  0.00000  0.17111
      12     H     0.42538  0.42538  0.00000  0.14924
      13     H     0.41294  0.41294  0.00000  0.17412
      14     H     0.42054  0.42054  0.00000  0.15892
      15     H     0.42050  0.42050  0.00000  0.15901
      16     H     0.41327  0.41327  0.00000  0.17347
      17     H     0.41816  0.41816  0.00000  0.16368
      18     H     0.41925  0.41925  0.00000  0.16150

   Total alpha = 25.00000, Total beta = 25.00000, Total charge = -0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
